Sources/Charcoal/Filters/List/ListFilterViewController.swift

```diff
@@ -78,6 +83,10 @@ public final class ListFilterViewController: FilterViewController {
 
     public override func showBottomButton(_ show: Bool, animated: Bool) {
         super.showBottomButton(show, animated: animated)
+        if show {
+            tableViewBottomAnchorConstraint.constant = -bottomButton.frame.height
```

Comment:
@finn-no/ios This is the last PR that needs a review before search filter tags are done! I would especially like feedback on my layout fixes in this file. I'm not 100% sure if it's safe, but didn't find another way to solve it. Read the PR description for more info 😄